buildsystem-core/src/main/java/de/eintosti/buildsystem/world/WorldManager.java

```diff
@@ -668,6 +668,8 @@ public boolean isPermitted(Player player, String permission, String worldName) {
             return true;
         }
 
+        if(buildWorld.getData().status().get() == WorldStatus.ARCHIVE) return false;
```

Comment:
This check isn't really smart because a lot of actions would be blocked by that. 
For example the status-change. With that logic you could put a world in Archive-Status but not back, due to a failed permission check. Or you can not delete archived worlds.
(unless you have admin permissions)

Reply:
I know that's why I did that